bcgov/House-Innovations-HOUSNAV
# FE-5: Header Component

**Purpose**
- Create a reusable header component to display on all pages of the web application
- Includes mobile, tablet, and desktop displays

**Testing**
1. See header on home page
2. Click to walkthrough and see same header
3. Click "Home" in header to see home page
4. Click "Download" in header to download the full PDF of the building code
5. Switch to mobile device OR resize browser to see different screen size displays
6. Open mobile navigation and see same links as on desktop

**Notes**
The "Download" link currently points to the hosted PDF file on BC Govs website. Do we think this is sufficient?
